images/views.py

```diff
@@ -13,7 +13,7 @@ class BannerRetrieveUpdate(RetrieveUpdateAPIView):
     permission_classes = (UserIsProfileOwnerOrReadOnly,)
     serializer_class = BannerSerializer
     parser_classes = (MultiPartParser, FormParser)
-    queryset = Profile.objects.all()
+    queryset = Profile.objects.filter(is_deleted=False, person__is_active=True)
```

Comment:
Such stuff fits nicely into managers (google/investigate, if you're not aware about it).

With managers, you can do stuff like `Profile.objects.only_active().filter(...)` (where `.only_active()` parts applies some common filters).